dockertest/subtestbase.py

```diff
@@ -168,6 +172,21 @@ def failif_not_in(needle, haystack, description=None):
                              (expect_s, description, haystack))
 
     @classmethod
+    def failif_not_redhat(cls, xcept=DockerTestNAError):
+        """
+        Raises an exception if this is a non-RHEL/RHELAH System
+
+        :param xcept: Non-default exception to raise
+        :raise DockerTestNAError: By default
+        """
+        try:
+            with open(cls.redhat_release_filepath, 'rb') as rhrel:
+                if 'Enterprise' not in rhrel.read():
+                    raise xcept("Expecting a Red Hat Enterprise Linux System")
+        except IOError:
+            cls.logwarning("Problem checking /etc/redhat-release")
```

Comment:
Is this code ever going to run on non-{RH,CentOS,Fedora}? If so then yes, raising `TestNAError` is probably the right thing to do.